docs/stake/nft_guide/nft_guide.md

```diff
@@ -0,0 +1,276 @@
+
+    />
+
+     </div>
+
+**Note:** NFTs are sold and transferred on the Fluence Network. To be able to view the NFT in their wallet, the recipient must first add the Fluence Network. Read more on preparing the wallet to work with Fluence Network in the corresponding [guide](../wallets_guide/wallets_guide.md).
+
+### Set NFT for sale
+
+When you list an NFT for sale, and someone buys it, the funds are automatically transferred to your address. To sell an NFT, open the NFT card to be sold and follow these steps:
+
+1.  Approve the Marketplace to manage all your NFTs - to transfer them automatically to a buyer; if you haven’t done it before - press the “Approve collection” button.
+
+     <div style={{ textAlign: "center" }}>
+
+    <img
+
+          src={require('./assets/nft_card_approve_collection.png').default}
+          alt="The view of the NFT card with the approve collection button."
+          style={{ display: "block", margin: "auto", maxWidth: "80%" }}
+
+    />
+
+    </div>
+
+2.  Confirm the approval in your wallet.
+
+    <div style={{ textAlign: "center" }}>
+
+    <img
+
+          src={require('./assets/nft_approve_collection_wallet.png').default}
+          alt="The view of the wallet window to confirm the NFT collection approval."
+          style={{ display: "block", margin: "auto", maxWidth: "80%" }}
+
+    />
+
+    </div>
+
+3.  Set your desired selling price for the NFT. Press the "Set to Sell" button and confirm the transaction in your wallet.
+
+    <div style={{ textAlign: "center" }}>
+
+    <img
+
+          src={require('./assets/confirm_sell_nft_wallet.png').default}
+          alt="The view of the wallet window to confirm the NFT sale."
+          style={{ display: "block", margin: "auto", maxWidth: "80%" }}
+
+    />
+
+    </div>
+
+4.  Check that the listed NFT has the correct price.
+5.  Wait for the NFT to be sold to someone. Once sold, you'll automatically receive your tokens.
+
+:::note
+
+Even if your NFT is set for sale, you still can transfer it manually without changing the “for sale” status. To do so, open its card and follow the steps described above in the “Transfer NFT” section.
+
+:::
+
+### How to Change the price or Delist an NFT from a sale
+
+You can easily adjust the price or remove an NFT you've listed for sale. Click on the card of the NFT on sale and choose your action:
+
+- To remove the NFT from sale, press the "Delist" button and confirm the transaction in your wallet.
+- To change the price, enter the new amount, press the "Change price" button, and confirm the transaction in your wallet.
+
+![Screenshot 2024-09-17 at 17.43.35.png](assets/nft_delist_button.png)
+
+<div style={{ textAlign: "center" }}>
+
+    <img
+
+          src={require('./assets/nft_delist_button.png').default}
+          alt="The view of the NFT card with the delist button."
+          style={{ display: "block", margin: "auto", maxWidth: "80%" }}
+
+    />
+
+    </div>
```

Comment:
same image twice. also I think this div wrapper around image tag is doing nothing, you can try removing it here and in other places. I understand you just don't like how images look with this syntax ![Screenshot 2024-09-17 at 17.43.35.png](assets/nft_delist_button.png) , right?